<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ThirdContainer">
    <item>
     <widget class="QLabel" name="thirdLabel">
      <property name="text">
       <string>Third</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="thirdComboBox"/>
    </item>
    <item>
     <widget class="QSpinBox" name="thirdSpinBox">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="FirstContainer">
    <item>
     <widget class="QLabel" name="firstLabel">
      <property name="text">
       <string>First</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="firstComboBox"/>
    </item>
    <item>
     <widget class="QSpinBox" name="firstSpinBox">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="SecondContainer">
    <item>
     <widget class="QLabel" name="secondLabel">
      <property name="text">
       <string>Second</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="secondComboBox"/>
    </item>
    <item>
     <widget class="QSpinBox" name="secondSpinBox">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="staticImages.qrc"/>
 </resources>
 <connections/>
</ui>
